doc/user_manual/chIntroduction.tex

```diff
@@ -103,9 +103,19 @@ \section{Evolution of SixTrack}
     \item \textbf{Version 4}
         \todo[inline]{Add stuff}
     \item \textbf{Version 5}
-        \todo[inline]{Add stuff}
+        Includes a full rewrite of the input parsing, upgrade of the source to Fortran 2008, and makes all major arrays dynamically allocatable.
```

Comment:
Put input parsing on a separate point, and not the very first thing. Also mention the splitting of the files, change to industry-standard preprosessing system, cmake.

Reply:
(I see astuce is mentioned already bellow, sorry I missed it)